### PR TITLE
Nested grouping and beta label removed from grouping

### LINKED ISSUE #3280: Nested grouping and beta label removed from grouping
## Description 

In 8.8, users can now create multi-level groups (nested grouping). Groups can be up to three levels deep and the grouping order (hierarchy) is determined by the selection order. 

The Beta label has also been removed in the UI, which means the feature is officially GA in 8.8. 

Related

- Epic: https://github.com/elastic/kibana/issues/150517 
- Issue: https://github.com/elastic/kibana/issues/150514
- Multi-level PR:: https://github.com/elastic/kibana/pull/152862
- Remove beta label PR: [https://github.com/elastic/kibana/pull/15701](https://github.com/elastic/kibana/pull/157015)

## Doc updates
- Remove the beta tag from the [Group alerts](https://www.elastic.co/guide/en/security/master/alerts-ui-manage.html#group-alerts) section.
- Include brief explanation of nested grouping in section intro para.

## Notes

- TBD

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #3280.

Preview: [Manage detection alerts | Group alerts](https://security-docs_3283.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#group-alerts)

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
lgtm!!

File: docs/detections/alerts-ui-manage.asciidoc [hunk 1]
```diff
@@ -41,12 +41,14 @@ image::images/additional-filters.png[Alerts table with Additional filters menu h
 [[group-alerts]]
 === Group alerts
 
-beta:[] You can group alerts by rule name, host name, user name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+You can group alerts by rule name, user name, host name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+
+Select up to three fields to create nested groups of alerts. The groups are nested in the order in which you select them.
```
Comment (benironside) [inline docs/detections/alerts-ui-manage.asciidoc:46]:
```suggestion
Select up to three fields by which to group alerts. The groups will nest in the order you selected them.
```

**Review by benironside — COMMENTED** [review]
LGTM, left a suggestion for your consideration

**Review by jmikell821 — APPROVED** [review]
LGTM!

File: docs/detections/alerts-ui-manage.asciidoc [hunk 2]
```diff
@@ -41,12 +41,14 @@ image::images/additional-filters.png[Alerts table with Additional filters menu h
...
-beta:[] You can group alerts by rule name, host name, user name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+You can group alerts by rule name, user name, host name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+
+Select up to three fields for grouping alerts. The groups will nest in the order you selected them.
 
 [role="screenshot"]
 image::images/group-alerts.png[Alerts table with Group alerts by drop-down]
 
-Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The type of information displayed varies depending on the selected field.
+Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The information displayed varies depending on the selected fields.
```
Comment (stephmilovic) [inline docs/detections/alerts-ui-manage.asciidoc:51]: Can we maybe describe how the selection order will determine the nesting order, and that the order is displayed in the "Group alerts by" label?

Comment (joepeeples) [reply docs/detections/alerts-ui-manage.asciidoc:51]: @stephmilovic We do mention the nesting order in the previous paragraph, above the screenshot, but I can add something about how the order is displayed in the label.

File: docs/detections/alerts-ui-manage.asciidoc [hunk 3]
```diff
@@ -41,12 +41,14 @@ image::images/additional-filters.png[Alerts table with Additional filters menu h
 [[group-alerts]]
 === Group alerts
 
-beta:[] You can group alerts by rule name, host name, user name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+You can group alerts by rule name, user name, host name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+
+Select up to three fields for grouping alerts. The groups will nest in the order you selected them.
```
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:46]:
```suggestion
Select up to three fields for grouping alerts. The groups will nest in the order you selected them, and the nesting order is displayed above the table next to *Group alerts by*.
```

**Pull request merged**